block/stream/output/token_service.proto

```diff
@@ -161,3 +164,31 @@ message UnpauseTokenOutput {}
  * the original transaction.
  */
 message UpdateTokenNftsOutput {}
+
+/**
+ * Block Stream data for a `tokenAirdrop` transaction.
+ *
+ * This message SHALL NOT duplicate information already contained in
+ * the original transaction.
+ */
+message TokenAirdropOutput {
```

Comment:
I do not think we need this.
The State Changes entries will list every created `PendingAirdrop`.
The transfer list will contain the custom fees.

Reply:
Just as with `CRYPTO_TRANSFER`, it is quite onerous to expect block stream consumers to be able to infer what part of the `TokenTransferList` is due to assessed custom fees; and we need to this recover the record stream.

For a top-level token airdrop, it's definitely possible to identify the created airdrops; but for multiple child airdrop transactions it would get tricky.

But maybe we can cross that bridge when we come to it... 🤔

Reply:
All of the state changes for a particular transaction follow that transaction in the stream.  It's definitely possible to connect the dots, where needed.
There is a bit of mild obsession in the BlockStream design with minimizing data because of the "eternal" cost associated with block stream storage that is never truly offset by one-time transaction fees for the transaction that creates that cost.

Reply:
Side note, I think we probably _should not_ try to recreate a record stream from a block stream.  It's very likely to be somewhere between extremely hard and impossible, in my opinion.

Block stream is very deliberately a different structure so that we can dramatically reduce data volume, unify several different streams, provide state proof services from block nodes instead of consensus nodes, and efficiently prove the authenticity of each block.

Reply:
Makes sense; removed the `new_pending_airdrops` field from this message.